src/Utils/Numbers.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/**
+ * This file is part of the Nette Framework (https://nette.org)
+ * Copyright (c) 2004 David Grudl (https://davidgrudl.com)
+ */
+
+declare(strict_types=1);
+
+namespace Nette\Utils;
+
+use Nette;
+
+
+/**
+ * Floats tools library.
+ */
+class Numbers
+{
+	use Nette\StaticClass;
+
+	/** @var float */
+	private static $epsilon = 1e-9;
+
+
+	public static function isZero(float $value): bool
+	{
+		return abs($value) < self::$epsilon;
+	}
+
+
+	public static function isInteger(float $value): bool
+	{
+		return abs(floor($value) - $value) < self::$epsilon;
+	}
+
+
+	public static function divide(float $dividend, float $divisor): float
+	{
+		if (self::isZero($divisor)) {
```

Comment:
Not sure how useful this is. Division by zero is well defined for floats. If you don't want to get `+Inf`, `-Inf` or `Nan` as a result, you would need to check more things.

Reply:
I don't know how much, but I did not encounter this problem. And I think, here is better to check at least zero than nothing.

If you have an example that would return NAN or INF, feel free to send it and try to fix it in that method.

Reply:
* `Nan / 3.0`
* `Inf` / 4.0